webpack.config.js

```diff
@@ -36,6 +36,7 @@ module.exports = (env = {}) => {
     new InlineEnvironmentVariablesPlugin('SCRIVITO_TENANT'),
     new CopyWebpackPlugin([
       { from: '../public' },
+      { from: '../src/intercom.js' },
```

Comment:
Why a copy, and not something like this https://github.com/Scrivito/scrivito_example_app_js/blob/master/webpack.config.js#L74 ?